src/patternfly/components/Card/examples/Card.md

```diff
@@ -509,6 +525,7 @@ A card is a generic rectangular container that can be used to build other compon
 | `.pf-c-card__actions` | `<div>` | Creates an actions element to be used in the card header. |
 | `.pf-c-card__header-main` | `<div>` | Creates a wrapper element to be used in the card header when using an image, logo, or text. |
 | `.pf-c-card__expandable-content` | `<div>` | Creates the expandable card's expandable content. |
+| `.pf-c-card__sr-input` | `<input>` | Creates an input which, when focused, makes a following `.pf-c-card` appear focused. |
```

Comment:
would this suggestion be accurate? I think "focused" would imply normal focus.

```suggestion
| `.pf-c-card__sr-input` | `<input>` | Creates an input which, when has screen reader focus, makes a following `.pf-c-card` appear focused. |
```

Reply:
Hmm, I think this is accurate as there's nothing about the class that makes the element inherently unfocusable.